xblock/fields.py

```diff
@@ -1013,22 +1042,22 @@ def scope_key(instance, xblock):
     if instance.scope.user == UserScope.NONE or instance.scope.user == UserScope.ALL:
         pass
     elif instance.scope.user == UserScope.ONE:
-        scope_key_dict['user'] = unicode(xblock.scope_ids.user_id)
+        scope_key_dict['user'] = str(xblock.scope_ids.user_id)
     else:
         raise NotImplementedError()
 
     if instance.scope.block == BlockScope.TYPE:
-        scope_key_dict['block'] = unicode(xblock.scope_ids.block_type)
+        scope_key_dict['block'] = str(xblock.scope_ids.block_type)
```

Comment:
^ This is why the *other* xblock conversion PR exists (#349): to surface the differences between a `six`-based approach, and a `future`-based approach, so we can better decide which one we want to advocate to the company.  

I think one of the issues here is that while futurize will do the initial conversions for you, it becomes easy to forget things during later development.  If I'm working in a file that doesn't currently use `str()`, and I add it, have I added the future import to the top of the file?  

Six requires more developer awareness and understanding of the differences up front, but is more explicit about where polyglot code lives.